Mathlib/Analysis/InnerProductSpace/Basic.lean

```diff
@@ -2230,11 +2230,13 @@ set_option linter.uppercaseLean3 false in
 #align real_inner_I_smul_self real_inner_I_smul_self
 
 /-- A complex inner product implies a real inner product -/
```

Comment:
Can you add a comment in the docstring explaining why it shouldn't be an instance?

Reply:
I added a comment. Thanks also for the remark about bench!